DiabloII.Application.Tests/Steps/Users/Create/CreateAUserSteps.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Threading.Tasks;
+using DiabloII.Application.Requests.Users;
+using DiabloII.Application.Tests.Apis.Users;
+using DiabloII.Application.Tests.Contexts.Users;
+using DiabloII.Application.Tests.Extensions;
+using TechTalk.SpecFlow;
+using TechTalk.SpecFlow.Assist;
+
+namespace DiabloII.Application.Tests.Steps.Users.Create
+{
+    [Binding]
+    [Scope(Tag = "users")]
+    public class CreateAUserSteps
+    {
+        private readonly IUsersApi _usersApi;
+      
+        private readonly IUsersTestContext _userContext;
+
+        public CreateAUserSteps(IUsersApi usersApi, IUsersTestContext userContext)
+        {
+            _usersApi = usersApi;
+            _userContext = userContext;
+        }
+
+        [Given(@"I create the users with the following informations")]
+        [When(@"I create the users with the following informations")]
+        public async Task WhenICreateTheUsersWithTheFollowingInformations(Table table)
+        {
```

Comment:
CreateUser / CreateSuggestion have the same base code, I should be able to extract this logic